Tests/ReactiveSwiftTests/FlattenSpec.swift

```diff
@@ -1011,6 +1011,42 @@ class FlattenSpec: QuickSpec {
 			}
 		}
 
+		describe("SignalProducer.concat(error:)") {
+			it("should emit concatenated error") {
+				let (signal, observer) = SignalProducer<Int, TestError>.pipe()
+
+				let mergedSignals = signal.concat(error: TestError.default)
+
+				var results: [Result<Int, TestError>] = []
+				mergedSignals.startWithResult { results.append($0) }
+
+				observer.send(value: 1)
+				observer.send(value: 2)
+				observer.send(value: 3)
+				observer.sendCompleted()
+
+				expect(results).to(haveCount(4))
+				expect(results[0].value) == 1
+				expect(results[1].value) == 2
+				expect(results[2].value) == 3
+				expect(results[3].error) == .default
+			}
+
+			it("should emit own error when present") {
```

Comment:
I'm not quite sure of the wording of this expectation. Thoughts?

Reply:
"should not concat the error", perhaps?

Reply:
I feel like that contradicts the earlier expectation - unless the reader looks at the implementation. However in my usage of Quick, I try to write expectations that could stand-alone without the implementation and still make sense.

Reply:
“should not concat a subsequent error if the earlier producer has failed”?

Reply:
"should not emit concatenated error to a failed producer"?

Reply:
Thank you for the help @andersio and @mdiep - I've settled on an alternative that I'm happy with.